waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/FederatedHMSHandler.java

```diff
@@ -148,14 +153,14 @@
 @Monitored
```

Comment:
Since this is now an abstract class shouldn't all these annotations be removed as they're on the non-abstract child class?